samples/snippets/src/main/java/com/example/asset/CreateFeedExample.java

```diff
@@ -27,15 +28,21 @@
 
 public class CreateFeedExample {
   // Create a feed
-  public static void createFeed(String[] assetNames, String feedId, String topic, String projectId)
-      throws Exception {
+  public static void createFeed(
+      String[] assetNames,
+      String feedId,
+      String topic,
+      String projectId,
+      ContentType contentType) throws Exception {
```

Comment:
Still need a more precise Exception here.

Reply:
Add in the last commit